### PR TITLE
Add/nft delegation docs

### PR DESCRIPTION
**Pull request by joshLong145**
# Description

Adds documentation for `Capacity Credits` and examples of
- how to mint a RLI NFT
- how to delegate usage of nft to another wallet

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Introducing new feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

Link to the relevant updated sections in the Netlify preview

- [ ] Link 1: [description of change]
- [ ] Link 2: [...]

# Checklist:

General
- [ ] I have performed a self-review of my code
- [ ] I have fixed all grammar issues (can use an AI tool to check), and explanations are in active voice
- [ ] I have checked the additions are concise
- [ ] Language is consistent with existing documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published (ie. SDK changes, node dependencies)

If I have added a new concept, I have
- [ ] included a beginner friendly explanation
- [ ] included a basic technical introduction and code sample
- [ ] new terms are defined, both in relevant new pages and in the glossary

### CODE REVIEW

File: docs/concepts/rate-limit-nft-concept.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+# **The Rate Limit Module (RLM)**
+
+In order to send transactions on Lit, you must first authenticate with the [Lit nodes](https://developer.litprotocol.com/v3/sdk/authentication/overview). This can be done using one of two ways:
+
+1. [Session signatures](https://developer.litprotocol.com/v3/sdk/authentication/session-sigs/intro): signatures scoped to specific capabilities or resources, designed to be ephemeral and limited in scope. (RECOMMENDED)
+2. [Auth sigs](https://developer.litprotocol.com/v3/sdk/authentication/auth-sig): a signature obtained from a user proving they own a particular a particular key (NOT RECOMMENDED)
+
+Every time you authenticate with Lit, the request context (i.e. wallet address, owned capacity credits, etc) is extracted and validated against the Rate Limiting Module to ensure capacity has not been breached.
+
+To mint a Capacity Credit NFT, you’ll need some testLITPRO tokens. These are test tokens that hold no real value and should only be used to pay for usage on Habanero. TestLITPRO tokens should only be claimed from the verified faucet, linked [here](https://faucet.litprotocol.com/).
```
Comment (a1ttech) [inline docs/concepts/rate-limit-nft-concept.md:35]: can we make sure to keep wording consistent? Here we say 'Capacity Credit NFT' and below 'Rate Limit NFT' is used

File: docs/concepts/rate-limit-nft-concept.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
+sidebar_position: 4
+---
+
+# Rate Limiting
```
Comment (a1ttech) [inline docs/concepts/rate-limit-nft-concept.md:5]: should we place this section somewhere else? Maybe under 'SDK Reference' with 'Paying for Usage' or under 'Network' ?

File: docs/concepts/rate-limit-nft-concept.md (hunk 1 above)
Comment (joshLong145) [reply docs/concepts/rate-limit-nft-concept.md:35]: Okay, should the wording be `Capacity Credits` or `Rate Limit NFT`
Also should we change the category bar to `Capacity Credits` or keep the `Rate Limit` wording?

File: docs/sdk/authentication/session-sigs/rate-limiting.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+In the above example, we are configuring 2 properties
+- `requestsPerDay` - How many requests can be sent in a 24 hour period.
+- `daysUntilUTCMidnightExpiration` - The number of days until the nft will expire. expiration will occur at `UTC Midnight` of the day specified.
+
+
+Once you mint your NFT you will be able to send X many requests per day where X is the number specified in `requestsPerDay`.
+
+:::note
+To use your new Rate Limit Increase NFT you will have to sign an `Auth Signature` with the the wallet which holds the NFT.
```
[the remark this answers is not in the document]
Comment (joshLong145) [reply docs/sdk/authentication/session-sigs/rate-limiting.md:40]: see above

File: docs/concepts/rate-limit-nft-concept.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+# Overview
+
+By default, all users get three free requests on Lit every 24 hours. In order to use the network beyond the rate limit, you must reserve additional capacity. This can be done using capacity credits, which allow holders to reserve a configurable number of requests (measured in requests per second) over a fixed length of time (i.e. one week).
+
+:::note
+Currently Rate Limiting is only enabled on `Habanero` and `Manzano`
+see [here](https://developer.litprotocol.com/v3/network/networks/testnet) for test networks
```
Comment (a1ttech) [inline docs/concepts/rate-limit-nft-concept.md:13]: for links, do we want to redirect or use local references? (i.e. /v3/network... vs https://developer...)

File: docs/concepts/rate-limit-nft-concept.md (hunk 2 above)
Comment (joshLong145) [reply docs/concepts/rate-limit-nft-concept.md:5]: I am not sure, open to suggestions.

File: docs/concepts/rate-limit-nft-concept.md (hunk 1 above)
Comment (a1ttech) [reply docs/concepts/rate-limit-nft-concept.md:35]: personally in favor of Capacity Credits

Comment (a1ttech) [reply docs/concepts/rate-limit-nft-concept.md:35]: for category name, i'd say 'Paying for Usage'

File: docs/concepts/rate-limit-nft-concept.md (hunk 2 above)
Comment (a1ttech) [reply docs/concepts/rate-limit-nft-concept.md:5]: let's move this to withib the SDK Reference section under 'Paying for Usage'

File: docs/concepts/rate-limit-nft-concept.md (hunk 1 above)
Comment (joshLong145) [reply docs/concepts/rate-limit-nft-concept.md]: updated

**Pull request merged**